apps/myjobs/app/assets/javascripts/workflows.js.coffee

```diff
@@ -103,6 +103,11 @@ $(window).focus ->
     update_missing_data_cluster_view()
     update_missing_data_path_view()
     update_missing_data_script_view()
+    if(data.xdmod_url)
+      # details external url
+      $("#job-details-id").html('<a target="_blank" href="'+data.xdmod_url+'">'+data.pbsid+' - <i style="display: inline" class="fa fa-external-link-square-alt"></i>&nbsp;XDMoD</a>')
+    else
+      $("#job-details-id").text(data.pbsid)
```

Comment:
will pbsid work for other schedulers?

Reply:
Yeah that is a legacy name for the job id when we were only working with torque. Since its difficult to do database migrations we just stuck with it :-).